packages/types/src/appearance.ts

```diff
@@ -564,6 +564,12 @@ export type Layout = {
    * @default undefined
    */
   logoImageUrl?: string;
+  /**
+   * This option pass the logoLinkUrl value to logo href.
+   * By default, the logo url will be the Home URL you've set in the Clerk Dashboard.
+   * Clicking the link will redirect to passed URL
+   */
```

Comment:
```suggestion
  /**
   * Controls where the browser will redirect to after the user clicks the application logo, usually found in the SignIn and SignUp components. If a URL is provided, it will be used as the `href` of the link.
   * @default Clicking clicking the application logo will redirect to Home Url, as set in the Clerk Dashboard 
   */
```